packages/angular_devkit/schematics/tasks/tslint-fix/executor.ts

```diff
@@ -183,7 +183,10 @@ export default function(): TaskExecutor<TslintFixTaskOptions> {
         const formatter = new Formatter();
 
         const output = formatter.format(result.failures, result.fixes);
-        if (output) {
+        // Certain tslint formatters outputs '\n' when there are no failures.
+        // This will bloat the console when having schematics running refactor tasks.
+        // see https://github.com/palantir/tslint/issues/4244
+        if (output && output !== '\n') {
```

Comment:
`output && output.trim()` maybe? to handle all whitespace.

Reply:
Yeah, that's neater.